tools/minikube/scripts/start_pods.sh

```diff
@@ -1,47 +1,41 @@
 #!/bin/bash
 # For development purpose only
 # Set the environment variable for accessing your Automation Controller
-# export PINAKES_CONTROLLER_URL=<<your controller url>>
-# export PINAKES_CONTROLLER_TOKEN=<<your controller token>>
-# export PINAKES_CONTROLLER_VERIFY_SSL=True|False
 
 set -o nounset
 set -o errexit
 
+OVERRIDE_ENV_FILE="minikube_env_vars"
 PINAKES_CONTROLLER_URL=${PINAKES_CONTROLLER_URL:-}
 PINAKES_CONTROLLER_TOKEN=${PINAKES_CONTROLLER_TOKEN:-}
 PINAKES_CONTROLLER_VERIFY_SSL=${PINAKES_CONTROLLER_VERIFY_SSL:-}
 
-if [[ -z "${PINAKES_CONTROLLER_URL}" ]]; then
-  echo "Error: Environment variable PINAKES_CONTROLLER_URL is not set."
-  exit 1
-fi
-
-if [[ -z "${PINAKES_CONTROLLER_TOKEN}" ]]; then
-  echo "Error: Environment variable PINAKES_CONTROLLER_TOKEN is not set."
-  exit 1
-fi
-
-if [[ -z "${PINAKES_CONTROLLER_VERIFY_SSL}" ]]; then
-  echo "Error: Environment variable PINAKES_CONTROLLER_VERIFY_SSL is not set."
-  exit 1
-fi
-
-# Set the environment variable for accessing insights service
-# export PINAKES_INSIGHTS_AUTH_METHOD=certificate|credential
-# export PINAKES_INSIGHTS_TRACKING_STATE=True|False
-# export PINAKES_INSIGHTS_URL=<<your insights url>>
-
-# Check if metrics collection is turned on
-PINAKES_INSIGHTS_TRACKING_STATE=${PINAKES_INSIGHTS_TRACKING_STATE:-False}
-PINAKES_INSIGHTS_AUTH_METHOD=${PINAKES_INSIGHTS_AUTH_METHOD:-certificate}
-if [[ "${PINAKES_INSIGHTS_AUTH_METHOD}" == "certificate" ]]; then
-  PINAKES_INSIGHTS_URL=${PINAKES_INSIGHTS_URL:-https://cert.cloud.redhat.com/api/ingress/v1/upload}
-else
-  PINAKES_INSIGHTS_URL=${PINAKES_INSIGHTS_URL:-https://cloud.redhat.com/api/ingress/v1/upload}
-fi
-PINAKES_INSIGHTS_USERNAME=${PINAKES_INSIGHTS_USERNAME:-unknown}
-PINAKES_INSIGHTS_PASSWORD=${PINAKES_INSIGHTS_PASSWORD:-unknown}
+setup_env_vars() {
+    if [ ! -f "$OVERRIDE_ENV_FILE" ]; then
```

Comment:
If you run the script from the root project, which is the expected, this conditional fails. The checks doesn't happen and the controller's variables are not populated.

Reply:
Sorry, I have explained it wrong. This only can work the first time, but reading those envvars are useful, overriding those variables should happen always, and the conditional should be applied only to the copy command.

Reply:
@Alex-Izquierdo This is meant to be run only once for existing setup. We want users to be editing the minikube_env_vars in the root directory. This is to ease transition to the new file based env vars. In the minikube.md file we ask the user to create this file and edit this file. If someone doesn't read the doc we don't want to break their setup